scripts/deploy/utils/constants.ts

```diff
@@ -24,7 +24,7 @@ export const ETH_STRIKE_STEP = 100; // ETH strike prices move in increments of 1
 export const WBTC_STRIKE_STEP = 1000; // WBTC strike prices move in increments of 1000
 export const AAVE_STRIKE_STEP = 10;
 export const STRIKE_DELTA = 1000; // 0.1d
-export const PREMIUM_DISCOUNT = 800; // 0.80, 20% discount
+export const PREMIUM_DISCOUNT = 200; // 0.20, 80% discount
```

Comment:
the premium discount is the other way around remember. discount = 80 means 20% discount.